lib/histogram.js

```diff
@@ -45,13 +38,19 @@ class Histogram extends Metric {
 			return acc;
 		}, {});
 
-		this.bucketExemplars = this.upperBounds.reduce((acc, upperBound) => {
-			acc[upperBound] = null;
-			return acc;
-		}, {});
+		if (config.enableExemplars) {
+			this.enableExemplars = true;
```

Comment:
should we set this to `false` in the `else`? I know it was not done before, but still

Reply:
hi SimenB, `this.enableExemplars` already set to `false` in [line 27](https://github.com/siimon/prom-client/pull/606/files/5b58cd4f34ac3f2518a1f6daa5d861b883107329#diff-4cacad18c9a9f40e8b66ee6f58741215e06d9e1bcd886452f61e74daee8ea387R27)